cli/azd/pkg/input/testdata/Test_progressChangeHeader.snap

```diff
@@ -0,0 +1,78 @@
+<RL>header
+
+<RL><prefix>── title ───────────────────────
+
+<RL>
+<RL>
+<RL>
+<RL>
+<RL>
+
+<prefix>────────────────────────────────<RL><M1U><M1U><RL><M1U><RL><M1U><RL><M1U><RL><M1U><RL><RL>
```

Comment:
Agree with @weikanglim that I'm not sure if these snapshots add much value because I'm barely able to tell today that this is the correct output let along me 6 months from now.